common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
We have to be careful about adding additional metadata to this message for two reasons:
1. this is additional performance overhead on the critical path to getting an activation through the system
2. https://github.com/apache/incubator-openwhisk/issues/2118#issuecomment-292360083